### PR TITLE
riscv64: Enable syswraps for various syscalls

### PR DESCRIPTION
**Pull request by Xeonacid**
These syscalls are from tests of some software I came across.
None is `PLA` except `ptrace`. Have checked that none of the generic/linux-generic wrappers has architecture-specific stuff.
riscv64 don't have ptrace actions like `PTRACE_GET/SETREGS` or `PTRACE_GET/SETFPREGS` but only the generic ones on Linux. So it should be OK to use generic wrappers like what arm64 does.

Run clang-format on the two files BTW.

### CODE REVIEW

File: include/vki/vki-scnums-riscv64-linux.h [hunk 1]
```diff
@@ -34,133 +34,133 @@
 // is a #include of
 //      linux-5.10.4/include/uapi/asm-generic/unistd.h
 
-#define __NR_getxattr 8
```
Comment (petrpavlu) [inline include/vki/vki-scnums-riscv64-linux.h:37]: VKI (Valgrind Kernel Interface) files contain mostly copies of definitions from include files of a given target operating system. Upstream Valgrind keeps them generally formatted in the same way as they are present in the original files. I think it is done to simplify their future comparison. While I realize it is not really that important for these syscall numbers, it still looks sensible to me to follow that convention and be consistent with the upstream.

**Review by petrpavlu — COMMENTED** [review]
Added a minor comment about formatting of VKI files. Looks good to me otherwise, thanks.

File: include/vki/vki-scnums-riscv64-linux.h (hunk 1 above)
Comment (Xeonacid) [reply include/vki/vki-scnums-riscv64-linux.h]: Thanks for the review, fixed.

**Comment by petrpavlu** [comment]
Thanks!

**Pull request merged**